media/js/base/datalayer-page-id.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+if (typeof Mozilla == 'undefined') {
+    var Mozilla = {};
+}
+
+(function() {
+    var dataLayer = window.dataLayer = window.dataLayer || [];
+    var Analytics = {};
+
+    /** Returns page ID used in Event Category for GA events tracked on page.
+    * @param {String} path - URL path name fallback if page ID does not exist.
+    * @return {String} GTM page ID.
+    */
+    Analytics.getPageId = function(path) {
+        var pageId = document.getElementsByTagName('html')[0].getAttribute('data-gtm-page-id');
+        var pathName = path ? path : document.location.pathname;
+
+        return pageId ? pageId : pathName.replace(/\/[^\/]*/, '');
```

Comment:
Once you do this, adding an extra test to make sure a URL path that does not contain a locale functions as expected would be nice too.